data/tribes/initialization/atlanteans/units.lua

```diff
@@ -1675,7 +1849,12 @@ descriptions:new_tribe {
             -- TRANSLATORS: Purpose helptext for an atlantean military site: Guardhouse
             purpose = pgettext("atlanteans_building", "Garrisons soldiers to expand your territory."),
             -- TRANSLATORS: Note helptext for an atlantean military site: Guardhouse
-            note = pgettext("atlanteans_building", "If you’re low on soldiers to occupy new military sites, use the downward arrow button to decrease the capacity. You can also click on a soldier to send him away.")
+            note = pgettext("atlanteans_building", "If you’re low on soldiers to occupy new military sites, use the downward arrow "..
```

Comment:
@Noordfrees I think we talked about pure concatenation of sentences lately I think this needs changed back

Reply:
Since this is done within a single gettext call it'll work correctly:
```lua
_("foo ") .. _("bar")  -- Bug
_("foo " .. "bar")     -- Fine
```

There are several places in master where it's already done like this. However this makes it easy to get the whitespacing between the text fragments wrong, so I agree that it would be cleaner to use just one long string literal without concatenation.